### PR TITLE
Homework_10_Olha_Lesiuk

### PR DESCRIPTION
**Pull request by OlhaLesiuk**

### CODE REVIEW

File: Olha_Lesiuk/HW_10/task_1.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+class Toyota:
```
Comment (bohdandovbsyh) [inline Olha_Lesiuk/HW_10/task_1.py]: not okay. It should be a class Company. And I want to create any company I want. In your implementation it always be Toyota

File: Olha_Lesiuk/HW_10/task_2.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        The method is used to obtain worker's company
+        """
+        return self.__company
+
+    @property
+    def salary(self):
+        """
+        The method is used to obtain worker's salary
+        """
+        return self.__salary
```
Comment (bohdandovbsyh) [inline Olha_Lesiuk/HW_10/task_2.py:100]: I do not see setters and some additional methods. Please think more about this class.

File: Olha_Lesiuk/HW_10/task_1.py (hunk 1 above)
Comment (OlhaLesiuk) [reply Olha_Lesiuk/HW_10/task_1.py]: Fixed

File: Olha_Lesiuk/HW_10/task_2.py (hunk 2 above)
Comment (OlhaLesiuk) [reply Olha_Lesiuk/HW_10/task_2.py:100]: Fixed

File: Olha_Lesiuk/HW_10/task_2.py [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+        The method is used to obtain worker's name
+        """
+        return self.__name
+
+    @name.setter
+    def name(self, name: str):
+        """
+        The method is used in case of changing to the new name
+        """
+        self.__name = name
```
Comment (bohdandovbsyh) [inline Olha_Lesiuk/HW_10/task_2.py]: I can set an empty name. It is not okay

File: Olha_Lesiuk/HW_10/task_2.py [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+        The method is used to obtain worker's gender
+        """
+        return self.__gender
+
+    @gender.setter
+    def gender(self, gender: str):
+        """
+        The method is used in case of changing gender
+        """
+        self.__gender = gender
```
Comment (bohdandovbsyh) [inline Olha_Lesiuk/HW_10/task_2.py]: i can set empty value

File: Olha_Lesiuk/HW_10/task_2.py [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+        The method is used to obtain worker's position
+        """
+        return self.__position
+
+    @position.setter
+    def position(self, position: str):
+        """
+        The method is used in case of promotion to the higher position
+        """
+        self.__position = position
```
Comment (bohdandovbsyh) [inline Olha_Lesiuk/HW_10/task_2.py]: again empty string can set

File: Olha_Lesiuk/HW_10/task_2.py [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+        The method is used to obtain worker's company
+        """
+        return self.__company
+
+    @company.setter
+    def company(self, company: str):
+        """
+        The method is used in case of changing to the new company
+        """
+        self.__company = company
```
Comment (bohdandovbsyh) [inline Olha_Lesiuk/HW_10/task_2.py]: same here

File: Olha_Lesiuk/HW_10/task_2.py [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+        The method is used to obtain worker's salary
+        """
+        return self.__salary
+
+    @salary.setter
+    def salary(self, salary: int):
+        """
+        The method is used in case of changing a salary
+        """
+        self.__salary = salary
```
Comment (bohdandovbsyh) [inline Olha_Lesiuk/HW_10/task_2.py]: I can set number less than 0

File: Olha_Lesiuk/HW_10/task_2.py [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+
+
```
Comment (bohdandovbsyh) [inline Olha_Lesiuk/HW_10/task_2.py:115]: please add some methods for this class

File: Olha_Lesiuk/HW_10/task_2.py (hunk 3 above)
Comment (OlhaLesiuk) [reply Olha_Lesiuk/HW_10/task_2.py]: fixed

**Review by bohdandovbsyh — APPROVED** [review]

**Pull request merged**